content/outcomes/application-development/balanced-teams.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Balanced Teams
+weight: 10
+layout: single
+team:
+  - VMware Tanzu Labs
+---
+
+Many teams today have chosen to follow an Agile approach to software development. Despite the proven benefits of such an approach, many of these teams still fail to realize their full potential. 
+
+One reason is that while these teams may place a priority on perfectly implementing their chosen Agile methodology, such as [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)), [XP](https://en.wikipedia.org/wiki/Extreme_programming), or [Kanban](https://en.wikipedia.org/wiki/Kanban_(development)), they neglect to consider the impacts to the human side of those methodologies. One of the most promising of these human practices is the balanced team.
+
+![Balanced Team Diagram](/images/outcomes/application-development/balanced-team.jpg)
+
+## What you will learn
+In this article, you will learn to:
+
+- [Define balanced team for your team members](#definition)
+- [Draw the balanced team diagram](#diagram)
+- [Identify the benefits of a balanced team](#benefits)
+
+## Define balanced team for your team members {id=definition}
+> “A team management philosophy that has people with a variety of skills and perspectives that support each other towards a shared goal.” - *balancedteam.org*
+
+The typical balanced team mix for product development is a product manager, product designer, and engineer(s) dedicated to a single product development effort. This is in contrast to another common team structure where roles some roles, such as product management and design, are shared across many product efforts.
+
+When we work within the construct of a balanced team, we ensure that all these perspectives blend and inform each other so that we build products that are desirable, usable, feasible, and viable.
+
+Our product development teams are:
+- [Small enough to be fed by two pizzas](http://blog.idonethis.com/two-pizza-team/) as pioneered by Amazon
+- Co-located (In person or remotely)
```

Comment:
remotely co-located seems like an oxymoron to me - should we just remove this  bullet?